views/peeps/peeps.erb

```diff
@@ -0,0 +1,8 @@
+<h1>Peeps</h1>
+<% if @peeps %>
```

Comment:
Don't make the view test the existence of `@peeps` - it should be the controller's responsibility to ensure this instance variable is available.